food.go

```diff
@@ -2,20 +2,30 @@ package main
 
 import "math/rand"
 
+const defpoints = 10
+
 type food struct {
 	emoji        rune
 	points, x, y int
 }
 
-func newFood(x, y int) *food {
+func newFood(x int, y int, emoji rune) *food {
 	return &food{
-		points: 10,
-		emoji:  randomFoodEmoji(),
+		points: defpoints,
+		emoji:  emoji,
```

Comment:
We don't wanna call this `emoji` anymore since it can be an `@` character now. Maybe just call it `ch` as short for character which is the same what `termbox-go` uses.